tests/unit/command/test_list.py

```diff
@@ -0,0 +1,14 @@
+from dvc.cli import parse_args
+from dvc.command.list import CmdList
+
+
+def test_list(mocker):
```

Comment:
Also need a functional test for `Repo.ls` API. Please be sure to add it to `tests/func/test_ls.py`